docs/source/dev_environment_setup.md

```diff
@@ -65,6 +65,8 @@ will try to find. Currently only `opendiff` is listed there (but we may add addi
 manually add it to this file. See the ApprovalTests [README](https://github.com/approvals/ApprovalTests.Python) for
 examples (illustrative particularly for file paths on Windows).
 
+!!! warning
```

Comment:
Hmm, at least on Github this warning is kinda lost at the bottom of the "Diff tool" subsection. How about moving this prominently towards the top of this file, or into its own sub-section?

Also, the `!!!` doesn't play nicely with Github's markup rendering. I assume it will displayed nicely in the rendered docs, but maybe we can find a way of displaying it which works well in both? Again, perhaps a subsection title including the word "warning" is a good compromise?

Reply:
Well, we use these elsewhere in the docs (they’re rendered by the admonition markdown extension). Is there much benefit in sticking to github flavoured markdown given these are for the docs site?

Reply:
Yeah you're right, probably not.